test/types/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "lib": ["es6", "dom"],
+    "noImplicitAny": false,
+    "noImplicitThis": false,
+    "strictNullChecks": false,
+    "strictFunctionTypes": false,
+    "esModuleInterop": true,
+    "types": []
```

Comment:
This is necessary to prevent the typescript compiler from pulling in and testing/linting node_modules when running `dts`, which it would do by default if this wasn't specified and cause this error https://github.com/microsoft/dtslint/issues/270